src/clims/services/application.py

```diff
@@ -24,6 +25,7 @@ def __init__(self):
         from sentry.plugins import PluginManager
         self.plugins = PluginManager(self)
         self.workflows = WorkflowService(self)
+        self.workbatches = WorkbatchService(self)
```

Comment:
In the PR I'll be presenting soon, the WorkBatches are all handled in the WorkflowService. I think that that makes sense because they are parts of the workflow. We can merge the two later.

Reply:
That sounds good for me. At this stage of work, I just needed a place to place my code.